### ISSUE TITLE
[Bug Report] popover met select not hidden

### ISSUE THREAD

**Issue by lightning0016** [ElemeFE/element#10927]
### Element UI version
2.3.6

### OS/Browsers version
all

### Vue version
2.5.17-beta.0

### Reproduction Link
[https://jsfiddle.net/wangxlliang/mmx38qxw/1866/](https://jsfiddle.net/wangxlliang/mmx38qxw/1866/)

### Steps to reproduce
popover  show  and click select

### What is Expected?
popover  show  and click select,popover should hide

### What is actually happening?
popover  show  and click select,popover not hide

**Comment by xianshenglu** [comment]
> `trigger="click"` means you have to click on the reference element (`<el-button>` in this case) to toggle its visibility. Use `trigger="manual"` if you need to hide the Popover when the button is blurred.

I don't think so. If users click the document, the popover would also hide. However, the click event in the select input was blocked by `@click.stop` in the select source code. That's why the popover wouldn't hide. @Leopoldthecoder

**Comment by xianshenglu** [comment]
Still exists in 2.13.1

**Comment by zhouxianjun** [comment]
I solved it like this:
Override popover's $el['@@clickoutsideContext'].documentHandler function.
Get the mouseup or mousedown event object, and then call popover.handleDocumentClick(mouseup or mousedown)

**Comment by appleqzw** [comment]
> I solved it like this:我是这样解决的： Add v-clickoutside instructions where popover is used 在使用弹出框的地方添加 v-click 外部说明 Override popover's $el['@@clickoutsideContext'].documentHandler function. 覆盖弹出框的 $el['@@clickoutsideContext'].documentHandler 函数。 Get the mouseup or mousedown event object, and then call popover.handleDocumentClick(mouseup or mousedown) 获取mouseup或mousedown事件对象，然后调用popover.handleDocumentClick(mouseup或mousedown)

It works.
